app/assets/stylesheets/application.css

```diff
@@ -1,2 +1,3 @@
 @import './global/*.css';
 @import '../../components/**/*.css';
+@import '../../views/**/*.css';
```

Comment:
@phoeinx @tillprochaska and I started a discussion on this, but didn't finish it.  My question was

> Component-only views - I see we have a PageHeader component, which I originally used, but it has a gray background and padding that I don't want. I added my own HTML, but that meant adding a dashboard/index.css file and importing it. From looking at the code, I think you don't do this anywhere, but favor a component-only solution. So, should I refactor the PageHeader to accept a gray/white background, with/without padding, etc?

He responded, and I responded, but then the conversation died. See the [Display notifications PR](https://github.com/tactilenews/100eyes/pull/1412), which was merged into this one.

I feel like we can discuss and add this as an improvement after this has been merged and deployed.

Reply:
See https://github.com/tactilenews/100eyes/issues/1454